ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4827,6 +4829,274 @@ HWRWISOVPM:
+  implements:
+  - MONITORING   
+  
+SSPM2X:
+  description: "Static pressure monitoring on the supply side where the average of the two sensors are being taken."
+  is_abstract: true
+  uses:
+  - supply_air_static_pressure_sensor_1
+  - supply_air_static_pressure_sensor_2
+  - supply_air_static_pressure_setpoint_1
+  - supply_air_static_pressure_setpoint_2   
+  implements:
+  - MONITORING
+
+SFRDM:  
+  description: "Supply fan run duration monitoring "
+  is_abstract: true
+  uses:
+  - supply_fan_run_time_accumulator
+  implements:
+  - MONITORING 
+
+CPRDM:  
+  description: "Compressor run duration monitoring "
+  is_abstract: true
+  uses:
+  - compressor_run_time_accumulator
+  implements:
+  - MONITORING 
+
+EVFSM:
+  description: "Evaporative fan speed monitoring ."
+  is_abstract: true
+  uses:
+  - evaporative_fan_speed_sensor
+  implements:
+  - CONTROL 
+
+RDM:
+  description: "Very basic system run duration monitoring. "
+  is_abstract: true
+  uses:
+  - run_time_accumulator
+  implements:
+  - MONITORING
+
+ZTLM:
+  description: "Zone temperature setpoint limit monitoring."
+  is_abstract: true
+  uses:
+  - high_limit_zone_air_temperature_setpoint
+  - low_limit_zone_air_temperature_setpoint
+  - zone_air_temperature_setpoint
+  implements:
+  - MONITORING
+  
+SFMSC:
+  description: "Discharge fan multi-speed control."
+  is_abstract: true
+  uses:
+  - supply_fan_run_command
+  - supply_fan_run_status
+  - supply_fan_speed_mode 
+
+SFRDM:
+  description: "Very basic system run duration monitoring for supply fans. "
+  is_abstract: true
+  uses:
+  - supply_fan_run_time_accumulator
+  implements:
+  - MONITORING  
+  
+#HRZAM:
+#  description: "High room Zone air temperature monitoring and control. "
+#  is_abstract: true
+#  uses:
+#  - high_limit_zone_air_temperature_setpoint
+#  implements:
+#  - CONTROL
+  
+#LRZAM:
+#  description: "Low room Zone air temperature monitoring and control. "
+#  is_abstract: true
+#  uses:
+#  - low_limit_zone_air_temperature_setpoint
+#  implements:
+#  - CONTROL  
+  
+ZHLM:
+  description: "Zone reative humidity setpoint limit monitoring."
+  is_abstract: true
+  uses:
+  - high_limit_zone_air_relative_humidity_setpoint
+  - low_limit_zone_air_relative_humidity_setpoint
+  - zone_air_relative_humidity_setpoint
+  implements:
+  - MONITORING
+
+#HRZHM:
+#  description: "High room Zone air humidity monitoring and control. "
+#  is_abstract: true
+#  uses:
+#  - high_limit_zone_air_relative_humidity_setpoint
+#  implements:
+#  - CONTROL    
+  
+#LRZHM:
+#  description: "Low room Zone air humidity monitoring and control. "
+#  is_abstract: true
+#  uses:
+#  - low_limit_zone_air_relative_humidity_setpoint
+#  implements:
+#  - CONTROL 
+
+
+CPRDM2X:  
+  description: "Compressor run duration monitoring of 2 compressors "
+  is_abstract: true
+  uses:
+  - compressor_run_time_accumulator_1
+  - compressor_run_time_accumulator_2  
+  implements:
+  - MONITORING  
+
+ZPLM:
```

Comment:
Are these sensors or setpoints? If setpoints, add the `zone_air_pressure_setpoint` to `uses` and adjust from `sensor` to `setpoint`.